### PR TITLE
Removes navigation settings that are not used by Volto

### PR DESCRIPTION
**Pull request by wesleybl**
fixes: #5961

### CODE REVIEW

File: packages/volto/src/config/ControlPanels.js [hunk 1]
```diff
@@ -90,6 +90,14 @@ export const filterControlPanelsSchema = (controlpanel) => {
       'picture_variants',
       'image_captioning',
     ],
+    navigation: [
+      'generate_tabs',
+      'navigation_depth',
+      'nonfolderish_tabs',
```
Comment (davisagli) [inline packages/volto/src/config/ControlPanels.js:96]: This one is used here: https://github.com/plone/plone.restapi/blob/main/src/plone/restapi/services/navigation/get.py#L116

Comment (wesleybl) [reply packages/volto/src/config/ControlPanels.js]: I removed it.

**Review by davisagli — APPROVED** [review]

File: packages/volto/src/config/ControlPanels.js (hunk 1 above)
Comment (wesleybl) [reply packages/volto/src/config/ControlPanels.js]: @davisagli even though `nonfolderish_tabs` is checked in restapi, its help text says:

```By default, any content item in the root of the portal will appear as a tab. If you turn this option off, only folders will be shown. This only has an effect if 'automatically generate tabs' is enabled.```

First, the term "tab" doesn't make sense in Volto. Second, the `automatically generate tabs` field has been removed. So, I think we should remove the nonfolderish_tabs field. Do you agree?

Comment (davisagli) [reply packages/volto/src/config/ControlPanels.js]: @wesleybl I see, so the help text doesn't match how it is actually used in plone.restapi. Yeah, I think we should probably remove it from the control panel.

**Pull request merged**